modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -10,6 +10,8 @@ any-category=Any Category
 any-folder=Any Folder
 any-site=Any Site
 any-user=Any User
+asset-tag-search-facet-display-context=Tag Facet Display Context
+asset-tag-search-facet-display-context-help=Access the data contained in the AssetTagSearchFacetDisplayContext object (for example, its isNothingSelected returns true if terms have not been selected).
```

Comment:
Grammar seems weird?  

Should it read:
(for example, ~~its~~ isNothingSelected returns true if terms have not been selected).

Reply:
I think there was another example of that weird grammar in another pull @thektan sent too.  Can you double check the other "help" text to make sure they sound ok?  Thanks!

Reply:
Will do, thanks Jon!